src/App.js

```diff
@@ -6,10 +6,11 @@ import Content from './components/layouts/Content';
 import { BrowserRouter } from 'react-router-dom';
 
 function App() {
+  let color="black";
   return (
     <div>
       <BrowserRouter>
-      <Header />
+      <Header color={color}/>
```

Comment:
I see you have passed the color but not used anywhere, can we delete it if not used or use it on the Header